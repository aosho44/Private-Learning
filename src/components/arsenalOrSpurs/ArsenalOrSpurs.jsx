import { useState } from 'react';

function ArsenalOrSpurs() {
	const [team, setTeam] = useState('');

	function Arsenal() {
		setTeam('Good choice you are very wise');
	}
	function Spurs() {
		setTeam('Horrible choice. Second chance');
	}
	function Clear() {
		setTeam('');
	}

	return (
		<div>
			<button onClick={Arsenal}>Arsenal</button>
			<br />
			<button onClick={Spurs}>Spurs</button>
			<br />
			<button onClick={Clear}>Clear</button>
			<h1>{team}</h1>
		</div>
	);
}
export { ArsenalOrSpurs };

// conditional operators: condition ? exprIfTrue : exprIfFalse
