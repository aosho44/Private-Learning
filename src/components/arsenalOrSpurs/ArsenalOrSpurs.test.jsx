import renderer from 'react-test-renderer';
import { ArsenalOrSpurs } from './ArsenalOrSpurs';

it('Checks if it displays Arsenal or Spurs descriptions when buttons are clicked', () => {
	const tree = renderer.create(<ArsenalOrSpurs />).toJSON;
	expect(tree).toMatchSnapshot();
});
