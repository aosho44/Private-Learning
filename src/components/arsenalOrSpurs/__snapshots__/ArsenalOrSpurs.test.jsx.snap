// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checks if it displays Arsenal or Spurs descriptions when buttons are clicked 1`] = `[Function]`;
